include/iscsi_if.h

```diff
@@ -104,6 +104,10 @@ enum iscsi_uevent_e {
 enum iscsi_tgt_dscvr {
 	ISCSI_TGT_DSCVR_SEND_TARGETS	= 1,
 	ISCSI_TGT_DSCVR_ISNS		= 2,
+	/*
+	 * SLP Discovery has never been implemented,
+	 * but this is kept for compatability with kernel code
+	 */
```

Comment:
I would not change this, because it's supposed to mirror the kernel's iscsi_if.h. There is only minor compat changes to allow building it different userspace setups.

Reply:
I will remove that comment, as a new commit. If you'd prefer I resubmit the branch, instead, let me know.